### ISSUE TITLE
Reading and converting data from other geological modelling packages

### ISSUE THREAD

**Issue by endarthur** [cgre-aachen/gemgis#86]
It would be nice to be able  to read data exported or contained in file formats specific for other modeling packages, such as Leapfrog Geo and Datamine.

Leapfrog Geo's .msh format seems to be a simple ascii header + arrays in binary as described in the header, but it would be good to have some example files to check deeper into it. There seems to be some other formats associated with Leapfrog that would  need to be inspected as well.

As for Datamine/Studio RM, in the very least there is a nice description of the .dm file online:
[http://www.vmine.net/vmine/dmformat.asp](http://www.vmine.net/vmine/dmformat.asp)

It's not particularly hard to parse, though there might be some extra steps in interpreting the data, as it is effectively a generic table format.

We could check other formats as well, as required and as we are able to get example files.

**Comment by AlexanderJuestel** [comment]
Issue will be reopened if new data formats need to be parsed
